Railgun

musicmanvr opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
This PR introduces a new weapon to the Syndicate arsenal for both traitors and nuclear operatives, the "Decimator" Railgun. It also includes a reverse engineered variant, admeme variant and laser variant for borgs when that system is ready for use, similar to the laser L6. Reverse engineering also unlocks the non-lethal bolts, the standard one must be reverse engineered.

The Decimator has a long charge-up, must be wielded, and shoots extremely slowly (0.30 firerate, 0.40 reverse engineered), but does 110 damage spread across three different damage types with normal rods, 15 piercing, 75 blunt, and 20 heat, along with 80 stamina damage.

The railgun rods cannot be crafted, and must be obtained through; 
- The purchase of extra ammo for 2 telecrystals. 
- Reverse engineering in the sec techfab, 
- Or through purchasing of the bundle itself, which comes with three.

Bolts are retrievable and do not break, but can bounce.

The addition of this PR would give syndicate operatives a terrifying new tool for taking down lone targets, but its steep price and requirement to wield alongside the slow fire-rate leaves the user vulnerable. It is also very loud, and the sound of the metal rod impacting makes a distinct noise.

Slimepeople, Oni, and Diona can survive a shot due to their innate damage resistance.

Stats:
- 110 Base damage, 75 blunt, 15 Piercing, 20 Heat using Rebar Rods.
- Also deals 80 stamina damage on hit.
- Non-Lethal shots do 10 blunt and 100 stamina damage.
- 25% Slowdown when holding, 15% for reverse engineered variant.
- 0.3 Fire-rate, 0.4 for the reverse engineered variant.
- Gigantic size.
- Takes Rebar Rods, a new ammo type exclusive for the railgun.

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
Syndicate weaponry has been pretty much the same for a while now, and I wanted to bring some variety to their pool with a new type of weapon entirely. The railgun is a powerful choice for teams, decimates unarmored targets, and leaves lightly armored targets near death, but its slow fire-rate and limited ammo pool encourage caution. 
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
The railgun is coded entirely in yml, which makes it simple to make changes to. New sound effects and yml files have been added to the DeltaV section of the prototypes menu, and the weapon has its own syndicate uplink fill as well, locale file included. The initial code is a bit messy, but I've ironed out any errors that I found in my initial runs of testing.
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/DeltaV-Station/Delta-v/assets/54725557/c41db221-750d-4a5b-af12-de5e54e279a4

Reverse engineering makes the railgun craftable from the security techfab.

COST:

Railgun:
- 50 Steel
- 20 Glass
- 2.5 Gold
- 5 Silver
- 10 Plasma
- 5 Uranium

Rebar Rod:
- 3 Steel

Rebar Rod (Less Lethal):
- Steel: 2.5
- Plastic: 1

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->
:cl: - Mike and Still-Icarus
- add: Added the Decimator Railgun, a thunderous new weapon for both syndicate and nuclear operatives.

musicmanvr committed:
test

please work

musicmanvr committed:
cleanup

removing old assets causing issues

musicmanvr committed:
addition to uplink

both traitors and nukeops can buy this absurdly powerful cannon

musicmanvr committed:
Update railgun_ammo.yml

musicmanvr committed:
laser variant

shrimple as adding a new variant with two firemodes, overkill, and overkiller

musicmanvr committed:
laser railgun cleanup

fixing some trash

musicmanvr committed:
adds the caninator

shoot cans and scare security

musicmanvr committed:
admeme variant and removal of clown variant for now

musicmanvr committed:
Merge branch 'DeltaV-Station:master' into Railgun

musicmanvr committed:
Slowdown in Hands

Slows you down when any variant is held as to make the weapon more balanced

musicmanvr committed:
Merge branch 'Railgun' of https://github.com/musicmanvr/Delta-V into Railgun

musicmanvr committed:
Laser railgun Sprite

adds a unique sprite for the laser variant of the railgun

musicmanvr committed:
fix the sprite

i am dumb

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml:
```diff
@@ -9,3 +9,5 @@
       - id: BodyBag_Folded
       - id: Portafib
+
+
```
why are there two empty lines?

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun_ammo.yml:
```diff
@@ -0,0 +1,88 @@
+  parent: BulletRailgun
+  id: BulletRailgunAdmeme
+  name: Unholy Rod [Admeme]
+  description: By the time you're able to pick this up, the station's probably already destroyed.
```
suffix: DO NOT MAP

should be added, similar to the other admeme items however maybe i'm nitpicking

the name shouldn't have admeme, it's weird IC

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun_ammo.yml:
```diff
@@ -0,0 +1,88 @@
+    muzzleFlash: null
+  - type: Tag
+    tags:
+    - BulletRailgun
```
You haven't defined this tag anywhere;

should be done in prototypes/delta-v/tags.yml

DangerRevolution commented on Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml:
```diff
@@ -336,3 +336,15 @@
       - id: SyringeTranexamicAcid
       - id: SyringeHyronalin
+
+- type: entity
```
this should be in the appropriate area in the DeltaV namespace:
Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:
```diff
@@ -256,4 +256,46 @@
+  name: Decimator Laser
+  parent: BaseWeaponBattery
+  id: WeaponLaserRailgun
+  size: Huge
```
size isn't defined in the entity section
- type: Item
  sprite: DeltaV/Objects/Weapons/Guns/Railguns/railgun_laser.rsi
  size: Huge

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,139 @@
+#admeme variant
+
+- type: entity
+  name: Unmaker Railgun [Admeme]
```
as mentioned before, probably shouldn't be called [Admeme] IC, should have a [DO NOT MAP] suffix probably

leonardo-dabepis commented on Resources/Textures/DeltaV/Objects/Weapons/Guns/Railguns/railgun_laser.rsi/icon.png:
this sprite could use more shading similar to the red variant. Same with the other railgun laser sprite.

DangerRevolution commented on Resources/Textures/DeltaV/Objects/Weapons/Guns/Ammunition/Cartridges/railrod.rsi/meta.json:
```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Still-Icarus",
```
made by Still-Icarus (discord) or (github)

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,139 @@
+  - type: Wieldable
+  - type: GunRequiresWield
+    wieldSound:
+      path: /Audio/DeltaV/Effects/railgun_charge.ogg
```
this should come under - type: Wieldable

this is also defined in the parent: BaseWeaponRailgun, so is a duplicate anyways

leonardo-dabepis commented:
For weapons in general, but especially a weapon this powerful, I would prefer if there were inhand sprites so that people can have a visual indicator that they're about to get decimated.

DangerRevolution commented:
> For weapons in general, but especially a weapon this powerful, I would prefer if there were inhand sprites so that people can have a visual indicator that they're about to get decimated

+1 to this ^

DangerRevolution commented:
There's probably a lot more to be done in terms of parenting things better but that's best to leave to the end 🐙

musicmanvr committed:
Issue Cleanup

Dusts off some of the code and makes some balance changes, along with better parenting.

musicmanvr committed:
purchasable ammo and charge sound update

made the charge sound slightly less annoying and added ammo purchasable by both traitors and syndicate

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml:
```diff
@@ -0,0 +1,23 @@
+- type: entity
+  parent: BoxCardboard
+  id: RailgunAmmoBag
+  name: Rebar Box
```
nitpicking but should be "Box of Rebar" or "Rebar Ammo Box" or even "Box of Railgun Ammo" or "Railgun Ammo Box" idk smth

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:
```diff
@@ -119,7 +119,7 @@
   id: WeaponEnergyGunMini
   description: A light version of the Energy gun with a smaller capacity.
-  components: 
+  components:
```
random white space

musicmanvr replied:
im looking in the code and just straight up dont see this, am i crazy?

DangerRevolution replied:
i think you need to add a space after components: 
if that doesn't fix it it's just a random error that's fine to leave

DangerRevolution commented on Resources/Locale/en-US/deltav/store/uplink-catalog.ftl:
```diff
@@ -4,3 +4,9 @@ uplink-reinforcement-radio-nukie-mouse-desc = Calls in a specially trained mouse
 uplink-bionic-syrinx-implanter-desc = An implant that enhances a harpy's natural talent for mimicry to let you adjust your voice to whoever you can think of.
+# Bundles
+uplink-decimator-bundle-name = Decimator Bundle
+uplink-decimator-bundle-desc = The Decimator railgun and three extra rebar. Favored by Gorlex snipers.
```
this is nitpicking really, but uh, i don't know what the consensus is on adding lore bits like "Gorlex"; obviously a thing on 13 but is it a thing here?

musicmanvr replied:
the main reason i did this is for the flavor text that's on the hypospray and holoparasite descriptions, i didn't know it was frowned upon

DangerRevolution replied:
> the main reason i did this is for the flavor text that's on the hypospray and holoparasite descriptions, i didn't know it was frowned upon

👍 if it's on the holopara and hypospray it's fine, i didn't know it was 🤷

DangerRevolution replied:
i don't even know *if* it's something we want or not, was just nitpicking off of vibes alone

musicmanvr replied:
i mean i can change it if you want, it's no skin off my back

DangerRevolution replied:
Your PR, i'm a random guy suggesting changes, go with your feelings

deltanedas replied:
yeah its fine to use le lore

VMSolidus replied:
This is a fair place to use the name. I'll accept that.

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml:
```diff
@@ -0,0 +1,23 @@
+  parent: ClothingBackpackDuffelSyndicateBundle
+  id: ClothingBackpackDuffelSyndicateFilledRailgun
+  name: Decimator bundle
+  description: "Favored by Gorlex Maurauder snipers, this railcannon packs a punch. Comes with 3 shots."
```
same comment as above RE: Gorlex

DEATHB4DEFEAT replied:
```suggestion
  description: Favored by Gorlex Maurauder snipers, this railcannon packs a punch. Comes with 3 shots.
```

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,139 @@
+  name: Unmaker Railgun
+  parent: BaseWeaponRailgun
+  id: WeaponRailgunAdmeme
+  suffix: "Admeme"
```
should probably be DO NOT MAP similar to other admeme items

musicmanvr replied:
this is actually already changed, it should've gone through, but I guess it didn't

DangerRevolution commented on Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/hitscan.yml:
```diff
@@ -121,3 +121,18 @@
     sprite: Objects/Weapons/Guns/Projectiles/projectiles.rsi
     state: impact_blue
+
+- type: hitscan
```
Should be in DeltaV folder

musicmanvr committed:
QA pass

various little fixes to improve the code and general cleanup

musicmanvr commented:
WHAT THE FUCK DOES THIS ERROR MEEEEEEAN

DangerRevolution committed:
Merge branch 'master' into Railgun

Signed-off-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>

DangerRevolution commented:
@musicmanvr I've got two commits unpushed on my local branch, but i'm too scared and too tired to fuck around with them. check if you're missing something from these files BUT I have unfucked your PR, God willing it stays that way.

really sorry for everyone github bot has requested to review this

also i've royally fucked your master branch because i forgot to switch branches at first off your master :)

DangerRevolution committed:
error fixing

Signed-off-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml:
```diff
@@ -0,0 +1,23 @@
+  parent: BoxCardboard
+  id: RailgunAmmoBag
+  name: Box of Rebar
+  description: A box for storing metal rods, commonly used in construction.
```
Is this a RailgunAmmoBag or a Box?

musicmanvr committed:
Sprite Update + Reverse Engineering

This adds reverse engineering, the new sprites, and a host of balance changes.
- The Railgun now does 80 stamina damage again on hit with normal bolts.

musicmanvr committed:
Merge branch 'DeltaV-Station:master' into Railgun

musicmanvr committed:
Merge branch 'master' into Railgun

musicmanvr committed:
reverse engineered fast firerate

as a reward for researching new technology, you've made it slightly faster. yay. also resprites the non-lethal variant to look more plunger-like

musicmanvr committed:
Final Changes

ready for merge, sir!

musicmanvr commented:
With this last commit, the railgun is now functionally complete, since all checks are passing, I think it's ready.

metalgearsloth commented on Resources/Audio/DeltaV/Effects/license.txt:
```diff
@@ -1,3 +1,15 @@
+
+suction.ogg modified from
+https://pixabay.com/sound-effects/search/suction%20cup/
+liscensed under https://pixabay.com/service/terms/ as of 1/30/2024
```
Doesn't this fall under standalone use by distributing the content via github:

> You cannot sell or distribute the Content (either in digital or physical form) on a Standalone basis. This includes selling or distributing Content on a Standalone basis as an image, audio, video, NFT or other digital file (including through a stock media platform), as well as a print, wallpaper, poster or on merchandise or on other physical products.

Upstream we nuke pixabay assets entirely unless it falls under their old CC0 licence.

musicmanvr replied:
Sound effects have been replaced with ones that fall under creative commons free use licenses, there are better ways to handle copyright, but I made the changes anyways, you're welcome.

musicmanvr committed:
Copyright Update

Makes the sounds all under free use creative commons to avoid copyright issues.

musicmanvr committed:
Update license.txt

musicmanvr committed:
Merge branch 'master' into Railgun

musicmanvr committed:
remove old attributions

this sound has been altered as to not use pixabay

musicmanvr committed:
Merge branch 'DeltaV-Station:master' into Railgun

VMSolidus commented on Resources/Prototypes/DeltaV/Catalog/uplink_catalog.yml:
```diff
@@ -9,6 +9,49 @@
+  productEntity: ClothingBackpackDuffelSyndicateFilledRailgun
+  icon: { sprite: DeltaV/Objects/Weapons/Guns/Railguns/railgun.rsi, state: icon }
+  cost:
+    Telecrystal: 14
```
I'm not going to accept this PR for anything less than 21tc on a weapon that can 1-shot anyone with re-usable ammo. It's essentially a sidegrade to a China Lake with the 1-shot utility, and a vastly scarier Histrov. I'd rather it be set aside in the same class of weapons you can only get if you have an extra source for your TC. 21 base, 2 for the spare ammo.

musicmanvr replied:
danger kept seeding thoughts in my head that the one shot weapon for traitors was way overpriced so i'm glad to finally have a second pair of eyes :blunt:

21 is reasonable for traitors, for nukeops i think 20 is fine considering the reliability of other options

VMSolidus commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,151 @@
+    soundGunshot:
+      path: /Audio/Items/Toys/ian.ogg #barks so your ears are saved from BRRRRRRRRRRRRRRRR
+
+#clown variant, not finished
```
As a general rule of thumb, we've rejected clown themed joke items in the past. It's safe to say we likely wouldn't want a clown railgun.

musicmanvr committed:
price increase

sorry danger but 14 tc is way too low lmao, nukies get it for 20 tc

musicmanvr committed:
Merge branch 'master' into Railgun

Signed-off-by: Firewatch <54725557+musicmanvr@users.noreply.github.com>

VMSolidus reviewed (dismissed):
Thanks! It looks good now. I can't wait to get canoed by a man hiding in the bathroom with a rifle as big as he is tall.

musicmanvr committed:
Merge branch 'master' into Railgun

Signed-off-by: Firewatch <54725557+musicmanvr@users.noreply.github.com>

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml:
```diff
@@ -0,0 +1,23 @@
+  id: RailgunAmmoBag
+  name: Box of Rebar
+  description: A box for storing metal rods, commonly used in construction.
+  #prefix: Syndicate   # What is this meant to be for?
```
```suggestion
```
Prefixes show in the entity spawner, a bit of extra info for mappers or whoever

musicmanvr replied:
it's suffix :trollface:

musicmanvr replied:
anyways i did this

DEATHB4DEFEAT commented on Resources/Textures/DeltaV/Objects/Weapons/Guns/Projectiles/projectiles.rsi/meta.json:
```diff
@@ -0,0 +1,56 @@
+  "states": [
+    {
+      "name": "muzzle_beam_heavy_green",
+      "delays": [
+        [
+          0.060000002,
+          0.060000002,
+          0.060000002,
+          0.060000002,
+          0.060000002,
+          0.060000002,
+          0.060000002,
+          0.060000002
+        ]
+      ]
```
What's with the absurd accuracy?

musicmanvr replied:
it's the same as other laser projectiles

DEATHB4DEFEAT commented on Resources/Textures/DeltaV/Objects/Weapons/Guns/Projectiles/projectiles.rsi/meta.json:
```diff
@@ -0,0 +1,56 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Modified by Mike from cev-eris at https://github.com/discordia-space/CEV-Eris/tree/7ff8f28ea4de734f3cc3cb70f2d4e4b4263a988d",
```
Mike who? Where are they?

stellar-novas replied:
Mike is the pr author, @musicmanvr

DEATHB4DEFEAT replied:
Nobody else is going to know that from looking at the meta file

DEATHB4DEFEAT commented on Resources/Textures/DeltaV/Objects/Weapons/Guns/Ammunition/Cartridges/railrod_nonlethal.rsi/meta.json:
```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by Still-Icarus on Discord, modified by Mike",
```
Ditto; Mike

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/tags.yml:
Isn't this file supposed to be sorted alphabetically, like WizDen's?

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:
```diff
@@ -259,3 +259,46 @@
+# For Borgs
+
+- type: entity
+  name: Decimator Laser
```
```suggestion
  name: decimator laser
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml:
```diff
@@ -0,0 +1,23 @@
+- type: entity
+  parent: ClothingBackpackDuffelSyndicateBundle
+  id: ClothingBackpackDuffelSyndicateFilledRailgun
+  name: Decimator bundle
```
```suggestion
  name: decimator bundle
```

musicmanvr replied:
due to a renaming scheme this has been resolved

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml:
```diff
@@ -0,0 +1,23 @@
+- type: entity
+  parent: BoxCardboard
+  id: RailgunAmmoBag
+  name: Box of Rebar
```
```suggestion
  name: box of rebar
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+- type: entity
+  parent: BulletRailgun
+  id: BulletRailgunBasic
+  name: Rebar Rod
```
```suggestion
  name: rebar rod
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+- type: entity
+  parent: BulletRailgun
+  id: BulletRailgunReverseEngineered
+  name: Rebar Rod (Less Lethal)
```
```suggestion
  name: rebar rod (less lethal)
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+      - BulletRailgunReverseEngineered
+
+# Laser variant for borgs
+
```
```suggestion
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+
+
+# Fear that which cannot be understood
+
```
```suggestion
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+    state: beam_heavy_green
+  impactFlash:
+    sprite: DeltaV/Objects/Weapons/Guns/Projectiles/projectiles.rsi
+    state: impact_beam_heavy_green
```
This isn't an entity and shouldn't be in the Entities folder

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+- type: entity
+  parent: BulletRailgun
+  id: BulletRailgunAdmeme
+  suffix: "DO NOT MAP"
```
```suggestion
  suffix: DO NOT MAP
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+  id: BulletRailgunAdmeme
+  suffix: "DO NOT MAP"
+  name: Unholy Rod
+  description: By the time you're able to pick this up, the station's probably already destroyed.
```
```suggestion
  description: "By the time you're able to pick this up, the station's probably already destroyed."
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml:
```diff
@@ -0,0 +1,166 @@
+  parent: BulletRailgun
+  id: BulletRailgunAdmeme
+  suffix: "DO NOT MAP"
+  name: Unholy Rod
```
```suggestion
  name: unholy rod
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,105 @@
+    sprite: DeltaV/Objects/Weapons/Guns/Railguns/railgun_ert.rsi
+
+# admeme variant
+
```
```suggestion
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,105 @@
+  name: Unmaker Railgun
+  parent: BaseWeaponRailgun
+  id: WeaponRailgunAdmeme
+  suffix: "Admeme"
```
```suggestion
  suffix: Admeme
```

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,105 @@
+  parent: BaseWeaponRailgun
+  id: WeaponRailgunAdmeme
+  suffix: "Admeme"
+  description: So pants shittingly terrifying that it was banished to the bluespace dimension by Nar'Sie. It's back now.
```
What is this trying to say?

musicmanvr replied:
changed

DEATHB4DEFEAT commented on Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Railguns/railgun.yml:
```diff
@@ -0,0 +1,105 @@
+  parent: BaseWeaponRailgun
+  id: WeaponRailgunERT
+  description: A mighty railgun that packs a hefty punch.
+  suffix: reverse-engineered
```
```suggestion
  suffix: reverse engineered
```

VMSolidus committed:
Merge branch 'master' into Railgun

Signed-off-by: VMSolidus <evilexecutive@gmail.com>

musicmanvr committed:
Merge remote-tracking branch 'upstream/master' into Railgun

musicmanvr committed:
Merge remote-tracking branch 'upstream/master' into Railgun

musicmanvr committed:
requested changes

hopefully

musicmanvr committed:
conversion to mono

yipeeee

musicmanvr committed:
fix of that other fix for the new file rename shit

pure agony

musicmanvr committed:
proper noun

the Decimator is now a proper noun similar to other weapons

musicmanvr committed:
Update Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun.yml

Co-authored-by: DEATHB4DEFEAT <77995199+DEATHB4DEFEAT@users.noreply.github.com>
Signed-off-by: Firewatch <54725557+musicmanvr@users.noreply.github.com>

musicmanvr committed:
Update duffelbag.yml

musicmanvr committed:
Merge remote-tracking branch 'upstream/master' into Railgun

musicmanvr committed:
Update uplink_catalog.yml

musicmanvr committed:
Update rod_impact.ogg

musicmanvr committed:
Update uplink_catalog.yml

IamVelcroboy commented:
This is our second oldest PR, where are we at with this?

VMSolidus commented:
> This is our second oldest PR, where are we at with this?

Assuming all of me and Death's change requests were made, this is actually ready to merge. The test is a false negative related to mimic code RNG throwing off the map tests by complete random chance.

IamVelcroboy commented:
> The test is a false negative related to mimic code RNG throwing off the map tests by complete random chance.

Yep, very familiar with that one. Haha

deltanedas committed:
Merge branch 'master' into Railgun

IamVelcroboy commented:
@DEATHB4DEFEAT does the change you requested still need to be addressed?

DannyFranklin reviewed (commented):
Finesse
